package.json

```diff
@@ -41,38 +41,38 @@
   },
   "dependencies": {
     "@a11y/focus-trap": "git+https://github.com/patrickarlt/focus-trap.git#conditional-define-build",
-    "@popperjs/core": "^2.0.6"
+    "@popperjs/core": "~2.1.1"
   },
   "devDependencies": {
     "@esri/calcite-base": "^1.2.0",
     "@esri/calcite-colors": "^2.1.0",
-    "@esri/calcite-ui-icons": "^3.0.0-beta.5",
-    "@stencil/core": "^1.8.9",
+    "@esri/calcite-ui-icons": "~3.0.0-beta.5",
+    "@stencil/core": "~1.11.2",
```

Comment:
this causes an error in ie11, all components in ie11 don't run since 1.10.X. We need to get stencil support on that. It's a system js error?

Reply:
Dang. I didn't know about that. I'll do a quick test w/ 1.9.x and will update this PR if it works. Otherwise, I'll hold off until the issue is fixed.

Reply:
Downgraded. 💥